pages.fr/android/am.md

```diff
@@ -0,0 +1,20 @@
+# am
+
+> Manager d'activité Android.
+> Plus d'informations : <https://developer.android.com/studio/command-line/adb#am>.
+
+- Commence une activité spécifique:
```

Comment:
Thank you for contribution. Every page in `page.fr` puts spaces in front of colons.
```suggestion
- Commence une activité spécifique :
```

Reply:
done ✅